src/System.Runtime/tests/System/Reflection/TypeInfoTests.netcoreapp.cs

```diff
@@ -17,7 +17,6 @@ private static IEnumerable<object[]> SZArrayOrNotTypes()
             yield return new object[] { typeof(int), false };
             yield return new object[] { typeof(int[]).MakeByRefType(), false };
             yield return new object[] { typeof(int[,]), false };
```

Comment:
It would be nice to move `MemberInfoTests.netcoreapp.cs` and `TypeInfoTests.netcoreapp.cs` over to System.Reflection as part of this since they tightly coupled with `MemberInfoTests.cs` and `TypeInfoTests.cs`.

Reply:
You also need to update `Configurations.props` but that should be all.

Also, the condition should better be `<ItemGroup Condition="'$(TargetGroup)' != 'netfx' And '$(TargetGroup)' != 'netstandard'">`.

Reply:
Done